.pre-commit-config.yaml

```diff
@@ -0,0 +1,63 @@
+# For more on configuring pre-commit hooks (see https://pre-commit.com/)
+
+# Force all unspecified python hooks to run python 3.8
+default_language_version:
+  python: python3
+
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.4.0
+  hooks:
+  - id: check-yaml
+    args: [--unsafe]
+  - id: check-json
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
+  - id: check-case-conflict
+- repo: https://github.com/psf/black
```

Comment:
Instead of pulling in a separate version of `black`, `flake8`, and `mypy`, and configuring them separately, we should run `hatch run lint:all && hatch run typecheck:all`.